### ISSUE TITLE
Select keyboard navigation does not work with submenus

### ISSUE THREAD

**Issue by stropitek** [zakodium-oss/react-science#622]
It kind of works - sub items can be selected only with the keyboard - but the submenus don't open when a subitem is active.

https://react-science.pages.dev/stories/?path=/story/forms-select--categories-nested
